packages/builder/src/contract/contracts/Xtokens/Xtokens.ts

```diff
@@ -1,59 +1,78 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import { AnyParachain, EvmParachain } from '@moonbeam-network/xcm-types';
 import { formatAssetIdToERC20 } from '@moonbeam-network/xcm-utils';
-import { isString, u8aToHex } from '@polkadot/util';
+import { u8aToHex } from '@polkadot/util';
 import { decodeAddress, evmToAddress } from '@polkadot/util-crypto';
 import { ContractConfigBuilder } from '../../ContractBuilder.interfaces';
-import { ContractConfig } from '../../ContractConfig';
+import { ContractConfig } from '../../../types/ContractConfig';
+import { XTOKENS_ABI } from './XtokensABI';
 
 const U_64_MAX = 18446744073709551615n;
+const XTOKENS_CONTRACT_ADDRESS = '0x0000000000000000000000000000000000000804';
```

Comment:
just as a heads up, there is an xTokens contract in Peaq EVM as well, with a different contract address, same for Batch

Reply:
I noted for a following PRs.